hpedockerplugin/request_validator.py

```diff
@@ -132,11 +132,6 @@ def _validate_import_vol_opts(self, contents):
                       (backend_name, backend_names)
                 raise exception.InvalidInput(reason=msg)
 
-            if config.replication_device:
```

Comment:
At this moment, we do not know if a replicated volume is created on the replicated backend. Therefore, we cannot check this condition here. Added this condition in volume_manager.py

Reply:
As we are llowing replicated backend to create non replicated volumes when rcg_name is not specified, we should allow legacy volume to be imported if this volume is non replicated